src/app/components/team/TeamReport.js

```diff
@@ -85,88 +86,98 @@ class TeamReportComponent extends React.Component {
           onChange={this.handleChangeLanguage.bind(this)}
         />
         <Can permissions={team.permissions} permission="update Team">
-          <p style={{ marginTop: units(2), textAlign: 'end' }}>
```

Comment:
Please remove units from import statement.

12:25 error 'units' is defined but never used no-unused-vars